docs/io/configuration/read_configuration.ipynb

```diff
@@ -4,12 +4,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Is this a sphinx note? Also typo recommend.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>